### DISCUSSION TITLE
Copilot x Educational pack

### DISCUSSION THREAD

**Discussion by SanLoveCode** [community/community#58099] [GitHub Education]
### Hi everyone,

I am verified with student educational pack, however, I can not use copilot for free, though the free q&a says that people who has student educational pack can use copilot for free. can anyone help me ?
![image](https://github.com/community/community/assets/69844040/0c0083a9-099d-42b9-a043-6001fb998c39)
![image](https://github.com/community/community/assets/69844040/7a6e427d-ff70-44f8-8253-2e743db637d2)
![image](https://github.com/community/community/assets/69844040/05e3e86d-33b8-428b-aff8-267cd0b0d610)

**Comment by iSaulX** [comment]
Hi SanLoveCode! As you said, you can use Copilot for free if you're student. There's two ways to claim it, the first, is go to [Home page](https://github.com) then go to to Settings > [billing](https://github.com/settings/billing) And scroll down to the sections of Add-ons, you'll see "You're eligible to use copilot for free" and then a button to claim it.

The second way is go to [Copilot page](https://github.com/features/copilot), login using your account which has the student status, sellect any plan and then click go "Get access" to copilot" (don't worry you won't be asked for any payment methods), after you'll see a message showing you "You're eligible to use copilot for free", and that's all. I hope you my answers help you. :)

Feel free to ask if you have another questions or anything else.

**Comment by samus-aran** [comment]
Hi @SanLoveCode ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
